From 6a0a4302336e653577ab6819c54e2bba9b2e1ef9 Mon Sep 17 00:00:00 2001
From: Calvin Johnson <calvin.johnson@nxp.com>
Date: Fri, 27 Oct 2017 11:20:47 +0530
Subject: [PATCH 06/22] staging: fsl_ppfe/eth: Make phy_ethtool_ksettings_get
 return void

Make return value void since function never return meaningful value

Signed-off-by: Calvin Johnson <calvin.johnson@nxp.com>

[ Srinidhi Rao : Ported this patch to photon linux from
  qoriq-components linux 'linux-4.14-nxp'
  (https://source.codeaurora.org/external/qoriq/qoriq-components/linux)
]

Signed-off-by: srinidhira0 <srinidhir@vmware.com>
---
 drivers/staging/fsl_ppfe/pfe_eth.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/staging/fsl_ppfe/pfe_eth.c b/drivers/staging/fsl_ppfe/pfe_eth.c
index a0171f3e7537..89e60ce424f0 100644
--- a/drivers/staging/fsl_ppfe/pfe_eth.c
+++ b/drivers/staging/fsl_ppfe/pfe_eth.c
@@ -577,7 +577,9 @@ static int pfe_eth_get_settings(struct net_device *ndev,
 	if (!phydev)
 		return -ENODEV;
 
-	return phy_ethtool_ksettings_get(phydev, cmd);
+	phy_ethtool_ksettings_get(phydev, cmd);
+
+	return 0;
 }
 
 /*
-- 
2.14.2

